config/docker/security/rotate-secrets.sh

```
@@ -0,0 +1,120 @@
+        rm "$tmp_json"
+    fi
+}
+
+# Function to rotate Vault token
+rotate_vault_token() {
+    local new_token=$(generate_random_string 64)
+    save_secret "$SECRETS_DIR/vault_token.txt" "$new_token"
+    
+    # Update environment variables
+    if [ -d "./env" ]; then
+        find ./env -type f -name ".env" -exec sed -i "s/VAULT_TOKEN=.*/VAULT_TOKEN=$new_token/" {} 
```

Shellcheck (0.10.0)

[error] Missing ';' or + terminating -exec. You can't use |/||/&&, and ';' has to be a separate, quoted argument.

(SC2067)